calcom/cal.com
# Settings: Tab item line height fix && Stop x-axis overflow of settings tab

## What does this PR do?

- Settings: Tab item line height fix 
- Stop x-axis overflow of settings tab



Fixes # (issue)
[Issue](https://github.com/calcom/cal.com/issues/4643)


**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?

- Test the settings tab to check the fixes i.e check the line-height of tab-item 
- Try scrolling horizontally on the settings tab
